packages/crayons-core/src/components/radio/radio.tsx

```diff
@@ -70,7 +70,10 @@ export class Radio {
           checked: true,
         });
       } else {
-        this.fwDeselect.emit();
+        this.fwDeselect.emit({
+          value: this.value,
+          checked: true,
```

Comment:
Should `checked` be `false` here?

Reply:
Yes correct Manoj, `checked` should be false. Have addressed in latest commit